documentation/cli/change_log.rst

```diff
@@ -8,6 +8,7 @@ since v0.12.0 | November XX, 2022
 =================================
 
 * Add ``--resolution``, ``--timezone`` and ``--to-file`` options to ``flexmeasures show beliefs``, to show beliefs data in a custom resolution and/or timezone, and also to save shown beliefs data to a CSV file.
+* Add options to ``flexmeasures add beliefs`` to 1) read CSV data with timezone naive datetimes (use ``--timezone``to localize the data), 2) read CSV data with datetime/timedelta units (use ``--unit datetime`` or ``--unit timedelta`, 3) remove rows with NaN values, and 4) add filter read-in data by matching values in specific columns (use ``--filter-column`` and ``--filter-value`` together).
```

Comment:
Just out of curiosity: I still wonder what the use case is where datetimes are in the event_value column.

Reply:
I had a case where I wanted to store arrival times and departure times as sensor data. That data does not map well to something like a binary availability sensor with a fixed sensor resolution, because arrivals and departures could happen at any time. And also, uncertainty about those values should be expressible in the time domain.

Reply:
Thanks!